public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/appInsightsGeoMap.json

```diff
@@ -0,0 +1,997 @@
+          "subscription": ""
+        },
+        "refresh": 1,
+        "regex": "([mM](icrosoft)\\.[iI](nsights)/(components))",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "current": {},
+        "datasource": {
+          "type": "grafana-azure-monitor-datasource",
+          "uid": "${ds}"
+        },
+        "definition": "",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Resource",
+        "multi": false,
+        "name": "res",
+        "options": [],
+        "query": {
+          "grafanaTemplateVariableFn": {
+            "kind": "ResourceNamesQuery",
+            "metricDefinition": "$ns",
+            "rawQuery": "ResourceNames($sub, $rg, $ns)",
+            "resourceGroup": "$rg",
+            "subscription": "$sub"
+          },
+          "queryType": "Grafana Template Variable Function",
+          "refId": "A",
+          "subscription": ""
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "current": {},
+        "datasource": {
+          "type": "grafana-azure-monitor-datasource",
+          "uid": "${ds}"
+        },
+        "definition": "",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Region",
+        "multi": true,
+        "name": "reg",
+        "options": [],
+        "query": {
+          "azureLogAnalytics": {
+            "query": "availabilityResults\r\n| distinct location",
+            "resource": "/subscriptions/$sub/resourceGroups/$rg/providers/$ns/$res"
+          },
+          "queryType": "Azure Log Analytics",
+          "refId": "A",
+          "subscription": ""
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "allValue": "",
+        "current": {},
+        "datasource": {
+          "type": "grafana-azure-monitor-datasource",
+          "uid": "${ds}"
+        },
+        "definition": "",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Availability Test",
+        "multi": true,
+        "name": "avTest",
+        "options": [],
+        "query": {
+          "azureLogAnalytics": {
+            "query": "availabilityResults\r\n| where location in ($reg)\r\n| distinct name",
+            "resource": "/subscriptions/$sub/resourceGroups/$rg/providers/$ns/$res"
+          },
+          "queryType": "Azure Log Analytics",
+          "refId": "A",
+          "subscription": ""
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "Availability test results count",
+          "value": "itemCount"
+        },
+        "hide": 2,
+        "includeAll": false,
+        "label": "Metric",
```

Comment:
I believe this variable is not necessary either, it's hidden (so users cannot modify it) and it's only being used for the title `"title": "${metric} (Sum)",`

Reply:
Also removed it! Thank you for all the feedback!